.buildkite/pipeline.yml

```diff
@@ -185,6 +185,19 @@
     - bazel test --config=ci $(./scripts/bazel_export_options) python/ray/new_dashboard/...
     - bazel test --config=ci $(./scripts/bazel_export_options) python/ray/serve/...
 
+- label: ":python: Minimal install"
+  conditions: ["RAY_CI_PYTHON_AFFECTED"]
+  commands:
+    - cleanup() { if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/travis/upload_build_info.sh; fi }; trap cleanup EXIT
+    - ./ci/travis/install-minimal.sh
+    - ./ci/travis/env_info.sh
+    - bazel test --test_output=streamed --config=ci $(./scripts/bazel_export_options)
```

Comment:
The conda environment installs some default dependencies, so it will not be an exact match. We can check if all default dependencies are installed (so what we have in setup.py is also installed in the machine), but I think that is very expected and not a check we need to have.

We might want to check that some additional dependencies are _not_ installed, but this should probably just be a few select dependencies and not the full list of packages (e.g. `starlette` for serve, `tensorboardX` for tune, `gym` for rllib, and `opencensus` for default). What do you think?

Reply:
Yeah, I think I mainly want to check for `default` tests.